CHANGELOG.md

```diff
@@ -1,288 +1,202 @@
+- [proxy] [\#5078](https://github.com/tendermint/tendermint/pull/5078) Fix a bug, where TM does not exit when ABCI app crashes (@melekes)
+- [evidence] [\#5170](https://github.com/tendermint/tendermint/pull/5170) change abci evidence time to the time the infraction happened not the time the evidence was committed on the block (@cmwaters)
+- [node] [\#5211](https://github.com/tendermint/tendermint/pull/5211) Don't attempt fast sync when the ABCI application specifies ourself as the only validator via `InitChain` (@erikgrinaker)
+- [libs/rand] [\#5215](https://github.com/tendermint/tendermint/pull/5215) Fix out-of-memory error on unexpected argument of Str() (@SadPencil)
+- [consensus] [\#5329](https://github.com/tendermint/tendermint/pull/5329) Fix wrong proposer schedule for validators returned by `InitChain` (@erikgrinaker)
+- [store] [\#5382](https://github.com/tendermint/tendermint/pull/5382) Fix race conditions when loading/saving/pruning blocks (@erikgrinaker)
+- [light] [\#5307](https://github.com/tendermint/tendermint/pull/5307) Persist correct proposer priority in light client validator sets (@cmwaters)
+- [docker] [\#5385](https://github.com/tendermint/tendermint/pull/5385) Fix incorrect `time_iota_ms` default setting causing block timestamp drift (@erikgrinaker)
+- [consensus] [\#5431](https://github.com/tendermint/tendermint/pull/5431) Check BlockParts do not exceed Maximum block size. (@cmwaters)
+- [mempool] [\#5483](https://github.com/tendermint/tendermint/pull/5483) Check tx size does not exceed available free space in the block. (@marbar3778)
+- [blockchain/v2] [\#5499](https://github.com/tendermint/tendermint/pull/5499) Fix "duplicate block enqueued by processor" panic (@melekes)
+- [abci/grpc] [\#5520](https://github.com/tendermint/tendermint/pull/5520) Return async responses in order, to avoid mempool panics. (@erikgrinaker)
+- [blockchain/v2] [\#5530](https://github.com/tendermint/tendermint/pull/5530) Fix "processed height 4541 but expected height 4540" panic (@melekes)
+- [consensus/wal] [\#5536](https://github.com/tendermint/tendermint/pull/5536) Fix WAL autorepair by opening target WAL in read/write mode (@erikgrinaker)
+- [block] [\#5567](https://github.com/tendermint/tendermint/pull/5567) Fix MaxCommitSigBytes (@cmwaters)
+- [blockchain/v2] [\#5553](https://github.com/tendermint/tendermint/pull/5553) Make the removal of an already removed peer a noop (@melekes)
+- [evidence] [\#5574](https://github.com/tendermint/tendermint/pull/5574) Fix bug where node sends committed evidence to peer (@cmwaters)
+- [privval] [\#5583](https://github.com/tendermint/tendermint/pull/5583) Make `Vote`, `Proposal` & `PubKey` non-nullable in Responses (@marbar3778)
+- [evidence] [\#5610](https://github.com/tendermint/tendermint/pull/5610) Make it possible for abci evidence to be formed from tm evidence (@cmwaters)
```

Comment:
this plays into evidence lifecycle refactor from above. It may be good to group them or have this be a sub bullet point

Reply:
Sure grouping them could make sense

Reply:
I've alphabetized everything - although it doesn't group everything together across sections, I think it makes this all easier to read (and to find related things)